Logger.Tests/LogFactoryTests.cs

```diff
@@ -1,11 +1,66 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
 using System.IO;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class LogFactoryTests
     {
-        
+        [TestMethod]
+        public void LogFactory_Constructor_Returns_Logger()
```

Comment:
In general the information you want to include in unit test method name is the method being tested, and any pre-conditions, and expectations. In this case "Constructor" is problematic since it is not the constructor that is being tested.

```suggestion
        public void CreateLogger_Returns_Logger()
```